src/lib.rs

```diff
@@ -93,13 +91,12 @@
 //! }
 //!
 //! {
-//!     // Use a read transaction to query the store by calling `Rkv.read()`
-//!     // to create a `Reader`.  There can be unlimited concurrent readers
-//!     // for a store, and readers never block on a writer nor other readers.
+//!     // Use a read transaction to query the store via a `Reader`.
+//!     // There can be unlimited concurrent readers for a store, and readers
```

Comment:
`Unlimited readers` is a bit misleading here, according to the [doc](https://github.com/LMDB/lmdb/blob/mdb.master/libraries/liblmdb/lmdb.h#L889), the default max_reader is 126, and could be configured by `lmdb-rs::env.set_max_readers()`.

Reply:
Ah, good point. In 0b805ff, I corrected this to say there can be multiple concurrent readers and then reference the default maximum number in a new documentation comment for `Rkv.read()`. (I didn't put the specific number in the crate documentation, as it's intended to be an introduction to the API, so I'd like to keep it simple.)

Reply:
Yep, that makes sense. As an aside, similar to `max_dbs`, we might want to expose it as a configurable in the environment builder in the future if needed (not likely for our use cases though).